### PR TITLE
ALBS-297: Added compression test logs

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

**Review by soksanichenko — APPROVED** [review]

File: alts/worker/runners/base.py [hunk 1]
```diff
@@ -369,14 +370,15 @@ def publish_artifacts_to_storage(self):
 
         def write_to_file(file_base_name: str, artifacts_section: dict):
             log_file_path = os.path.join(
-                self._artifacts_dir, f'{file_base_name}_{self._task_id}.log')
-            with open(log_file_path, 'w+t') as f:
-                f.write(f'Exit code: {artifacts_section["exit_code"]}\n')
-                f.write(f'Stdout:\n\n')
-                f.write(artifacts_section['stdout'])
+                self._artifacts_dir, f'{file_base_name}_{self._task_id}.gz.log')
```
Comment (Korulag) [inline alts/worker/runners/base.py:373]: `.gz` extension should be at the end

File: alts/worker/runners/base.py [hunk 2]
```diff
@@ -370,13 +371,14 @@ def publish_artifacts_to_storage(self):
         def write_to_file(file_base_name: str, artifacts_section: dict):
             log_file_path = os.path.join(
                 self._artifacts_dir, f'{file_base_name}_{self._task_id}.log')
-            with open(log_file_path, 'w+t') as f:
-                f.write(f'Exit code: {artifacts_section["exit_code"]}\n')
-                f.write(f'Stdout:\n\n')
-                f.write(artifacts_section['stdout'])
+            with open(log_file_path, 'wb') as file_:
```
Comment (soksanichenko) [inline alts/worker/runners/base.py:374]: just `fd` instead `file_`. I guess it's more suitable and short. fd - file descriptor

**Review by soksanichenko — APPROVED** [review]

**Review by maccelf — APPROVED** [review]

File: alts/worker/tasks.py [hunk 3]
```diff
@@ -95,7 +95,9 @@ def is_success(stage_data_: dict):
                      if task_params.get('repositories') else [],
                      'dist_arch': task_params.get('dist_arch')
                      if task_params.get('dist_arch') else 'x86_64'}
-
+    log_name = f"hui-{task_id}-{arch}.{ts}.log"
```
Comment (Korulag) [inline alts/worker/tasks.py:98]: Are these logs to be stored on the host? At some point they will result a "no space left"

**Review by Korulag — CHANGES_REQUESTED** [review]

File: alts/worker/tasks.py (hunk 3 above)
Comment (anfimovdm) [reply alts/worker/tasks.py:98]: that was a debug lines to help Maxim with his task and I added them by accident, I will remove it

**Pull request merged**